### PR TITLE
Update create profile and sign in screen

### LINKED ISSUE #21564: Onboarding - Update Create Profile screen
### Description

After pressing on "Create profile" in the new intro screen, the user should see the new "Create profile" screen. Notable changes:

- Friendlier wording in the main box title & subtitle (instead of the current "Generate keys" and "Create your new self-sovereign identity").
- Changed from "Use recovery phrase" to "Sign up with a seed phrase". We will try to avoid using the term "recovery phrase" because this is a terminology mostly used in Status.

### Out of scope

The new button "Sign up with an empty Keycard" shouldn't be implemented in this issue.

### Resources

- Figma https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15097-57508&node-type=frame&m=dev

### LINKED ISSUE #21565: Onboarding - New Sign in screen
### Description

From the new [Intro screen](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15075-43853&node-type=instance&m=dev), after pressing on "Sign in", we should make a few adjustments:

- Title "Sync or recover profile" changed to "Sign in".
- Change the main call to action button. Currently, it is "Sign in by syncing" and the secondary action is "Use recovery phrase". We are reversing the order, now, the main action should be to [Sign in with seed phrase](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15077-41768&node-type=frame&m=dev) and the [secondary call to action should be Sign in by syncing](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15077-41776&node-type=instance&m=dev).

### Out of scope

- Button [Sign in with Keycard](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15077-41778&node-type=instance&m=dev)

### Acceptance criteria

- Figma still has references saying "recovery phrase" such as [here](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15077-41773&node-type=instance&m=dev). Always use "seed phrase" from now on.
- As of Nov 5 2024, the new sign in screen in Figma doesn't have the [new asset for the CTA](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15078-41866&node-type=ellipse&m=dev). It's okay to use whatever image we have available now because we can update the asset in a follow-up PR.

### Resources

- Figma https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15122-56878&node-type=text&m=dev

### PR DESCRIPTION
**Pull request by Parveshdhull**
fixes https://github.com/status-im/status-mobile/issues/21565
fixes https://github.com/status-im/status-mobile/issues/21564

### Testing
Please update the e2e tests(if needed) and perform a light check for regressions in above screen.
**Note:** The content of the info buttons is out of scope.

status: ready

### CODE REVIEW

File: src/status_im/contexts/onboarding/create_or_sync_profile/view.cljs [hunk 1]
```diff
@@ -35,90 +36,115 @@
...
+  []
+  [quo/small-option-card
+   {:variant             :icon
+    :title               (i18n/label :t/use-a-recovery-phrase)
+    :subtitle            (i18n/label :t/use-a-recovery-phrase-subtitle)
+    :accessibility-label :use-a-recovery-phrase-icon-card
+    :image               (resources/get-image :ethereum-address)
+    :on-press            #(navigate-to-sign-in-by-recovery-phrase true)}])
+
+(defn- use-an-empty-keycard-icon-card
```
Comment (ilmotta) [inline src/status_im/contexts/onboarding/create_or_sync_profile/view.cljs]: Nitpick: usually articles are avoided in function names. Renamed would be `use-empty-keycard-icon-card` and `use-empty-keycard-icon-card`.

**Review by ilmotta — APPROVED** [review]
Just checking: @Parveshdhull, are the PNGs compressed?

**Review by flexsurfer — APPROVED** [review]

**Comment by Parveshdhull** [comment]
> Just checking: @Parveshdhull, are the PNGs compressed?

Thank you @ilmotta for reviewing the PR. Yes, PNGs are compressed.

**Comment by VolodLytvynenko** [comment]
hi @ilmotta I’m a bit unclear about the expected results and would like to clarify a few points if my verification is correct:

1. From this [original task](https://github.com/status-im/status-mobile/issues/21565) and this [design](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=12860-161699&node-type=text&m=dev) The description states:
> Title "Sync or recover profile" changed to "Sign in"

However, by [design](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15618-48102&node-type=instance&m=dev) I see "Log in" instead of "Sign in".  
The app currently matches the latter ("Log in").

Screen from the the latest build of current PR
image

2. In [original task](https://github.com/status-im/status-mobile/issues/21565#issue-2634940595) the description mentions:
> Figma still has references saying "recovery phrase" such as [here](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15077-41773&node-type=instance&m=dev). Always use "seed phrase" from now on.

However, in the new [design](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15077-41773&node-type=instance&m=dev) and in the latest build, "recovery phrase" is still used.
image

I understand the design might have been updated, which could cause discrepancies. To confirm:
Should I refer to the [simplified onboarding design](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15071-95173&node-type=canvas&m=dev) as a correct resource for verification?

**Comment by VolodLytvynenko** [comment]
hi @yevh-berdnyk Could you please adjust the e2e tests in this PR? It seems the accessibility ID use-recovery-phrase-option-card is no longer used. thanx!

**Comment by Parveshdhull** [comment]
Hi, @VolodLytvynenko, Thank you very much for taking the PR for testing.

Please use [latest design](https://www.figma.com/design/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=15071-95173&node-type=canvas&m=dev) (Simplified Onboarding) as source for comparison. Issue content is little outdated.

> Always use "seed phrase" from now on.

https://github.com/status-im/status-mobile/issues/21568#issuecomment-2489737978

**Comment by VolodLytvynenko** [comment]
@Parveshdhull thank you for clarification. One more question: Is it expected to see the "Feature not implemented" alert when selecting Use an empty Keycard during profile creation? From my discussion with @mariia-skrypnyk, it seems onboarding with an empty Keycard is available and enabled in the latest nightly. Could you clarify why it’s disabled in the PR?
image

**Comment by Parveshdhull** [comment]
Hi @flexsurfer, @ilmotta,

Is the implementation of the empty key-card flow complete? Also, should both `login with keycard` and `use empty keycard`  should use same dispatch, `:open-modal :screen/keycard.check`?

**Comment by yevh-berdnyk** [comment]
Hi @Parveshdhull, @VolodLytvynenko, E2E tests are updated, the recent failures are not PR related. Thanks!

**Comment by ilmotta** [comment]
> Is the implementation of the empty key-card flow complete? Also, should both `login with keycard` and `use empty keycard should use same dispatch, :open-modal :screen/keycard.check?

@Parveshdhull, @VolodLytvynenko, the PR to support the empty keycard flow is in progress https://github.com/status-im/status-mobile/pull/21623. @Parveshdhull I think it's up to you and @flexsurfer to decide who merges first and then the other adjust/rebase.

**Comment by Parveshdhull** [comment]
Thank you @ilmotta for confirming.

Hi @VolodLytvynenko, Since the 'Use Empty Keycard' flow is still under development, let's keep the 'Feature Not Implemented' popup for now. We can update it later as needed.

**Comment by VolodLytvynenko** [comment]
> Thank you @ilmotta for confirming.
> 
> Hi @VolodLytvynenko, Since the 'Use Empty Keycard' flow is still under development, let's keep the 'Feature Not Implemented' popup for now. We can update it later as needed.

@Parveshdhull agree. thank you for your work. PR can be merged!

**Pull request merged**